spec/requests/api/v1/users_spec.rb

```diff
@@ -1 +1,61 @@
+require 'rails_helper'
 
+RSpec.describe 'Api::V1::Users', type: :request do
```

Comment:
The structure of the tests is not accurate, we need to see easily in this file all the endpoints that we are testing. For example should be something like:
```
describe 'post #create' do
...
end

describe 'get #show' do
...
end
```